_data/simple-icons.json

```
@@ -6840,6 +6840,11 @@
             "hex": "17377F",
             "source": "https://mewe.com"
         },
+        {
+            "title": "Micro",
+            "hex": "2E3192",
+            "source": "https://github.com/zyedidia/micro/blob/master/assets/micro-logo-mark.svg"
```

```suggestion
            "source": "https://github.com/zyedidia/micro/blob/48645907ec55798b75723019dad75dba51bd97d7/assets/micro-logo-mark.svg"
```

Let's use the permalink instead.